### PR TITLE
Clarity on variables.  Minor formatting.

### PR DESCRIPTION
**Pull request by arnoldblinn**

### CODE REVIEW

File: Domain Connect Spec Draft.adoc [hunk 1]
```diff
@@ -555,11 +582,6 @@ template for a service that is sold through different companies. Not all templat
 |groupId
 |This OPTIONAL parameter specifies the group of changes from the template to apply. If no group is specified, all groups are applied. Multiple groups can be specified in comma delimited format.
 
-|*Provider Name*
```
Comment (pawel-kow) [inline Domain Connect Spec Draft.adoc]: Do we want to drop it? Wasn't that needed for Reseller case?

File: Domain Connect Spec Draft.adoc [hunk 2]
```diff
@@ -1547,43 +1595,66 @@ Such a template might contain an A record of the form:
...
 
-It is not allowed in any occasion that both _host_ and _pointsTo_/_data_ fields contain only variables. Such templates shall still define a constant part of _host_ or _pointsTo_/_data_ (a prefix) which will prevent phishing attack vectors which would impersonate
-the provider in order to push different data into DNS (which could be used to complete other mean of DNS-based verification).
+When %host% is not provided or defined, the use of @ in a template is equivalent to its use in the
+BIND notation as per RFC 1035.
 
 === Repository and Integrity
 
-This template format is intended largely for documentation and
+This template format is intended largely for documentation ands
```
Comment (pawel-kow) [inline Domain Connect Spec Draft.adoc]: Typo: additional s

File: Domain Connect Spec Draft.adoc [hunk 3]
```diff
@@ -52,9 +52,14 @@ toc::[]
 
 == Introduction and background
 
-GoDaddy recently implemented a feature called Domain Connect that simplified how Service Providers configured domains hosted at  GoDaddy (the DNS Provider). This made the end to end experience considerably easier for the end user.
```
Comment (pawel-kow) [inline Domain Connect Spec Draft.adoc]: Revision number and a date shall be bumped here before merging.

File: Domain Connect Spec Draft.adoc (hunk 1 above)
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: This was in the table twice.  It is still in there :-)

Comment (pawel-kow) [reply Domain Connect Spec Draft.adoc]: :ok:

**Pull request merged**